python-backend/app/api/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+
+import uuid
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import validates
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from . import permit_amendment_status_code, permit_amendment_type_code
+from ....utils.models_mixins import AuditMixin, Base
+
+
+class PermitAmendment(AuditMixin, Base):
+    __tablename__ = 'permit_amendment'
+    permit_amendment_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
+    permit_amendment_guid = db.Column(UUID(as_uuid=True))
+    permit_id = db.Column(db.Integer, db.ForeignKey('permit.permit_id'), nullable=False)
+    received_date = db.Column(db.DateTime, nullable=False, server_default=FetchedValue())
+    issue_date = db.Column(db.DateTime, nullable=False, server_default=FetchedValue())
+    authorization_end_date = db.Column(db.DateTime, nullable=False, server_default=FetchedValue())
```

Comment:
We have FetchedValue here, but below we enforce that they are set. do we allow the db to set the default, or does the python need to?